### PR TITLE
Update Identity API docs for Cordova

### PR DESCRIPTION
**Pull request by rymorale**

### CODE REVIEW

File: using-mobile-extensions/mobile-core/identity/identity-api-reference.md [hunk 1]
```diff
@@ -325,6 +325,73 @@ scheme://authority/path?TS=timestamp&MCMID=ecid&MCORGID=ecorgid@AdobeOrg#fragmen
...
+```text
+scheme://authority/path#fragment
+```
+
+The Adobe visitor data is appended as:
+
+```text
+scheme://authority/path?TS=timestamp&MCMID=ecid&MCORGID=ecorgid@AdobeOrg#fragment
+```
+
```
Comment (imironchuk) [inline using-mobile-extensions/mobile-core/identity/identity-api-reference.md:390]: Looks like this is missing the {% hint style="info" %} start tag.

File: using-mobile-extensions/mobile-core/identity/identity-api-reference.md [hunk 2]
```diff
@@ -872,6 +1027,49 @@ try {
...
 {% endtab %}
+
+{% tab title="Cordova" %}
+
+### getUrlVariables
+
+#### Cordova
+
+{% hint style="info" %}
+This method was added in react-native-acpcore v1.0.5.
```
Comment (imironchuk) [inline using-mobile-extensions/mobile-core/identity/identity-api-reference.md:1038]: is this reference to react-native correct?

File: using-mobile-extensions/mobile-core/identity/identity-api-reference.md (hunk 1 above)
Comment (rymorale) [reply using-mobile-extensions/mobile-core/identity/identity-api-reference.md:390]: The hint start tag is at line 366. I did spot an issue with the link at 391 which I will fix.

File: using-mobile-extensions/mobile-core/identity/identity-api-reference.md (hunk 2 above)
Comment (rymorale) [reply using-mobile-extensions/mobile-core/identity/identity-api-reference.md:1038]: whoops, thanks for catching this. i'll fix it as well.

**Pull request merged**